modules/virt-viewing-outdated-workloads.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-viewing-outdated-workloads_{context}"]
+= Viewing outdated {VirtProductName} workloads
+
+If there are outdated virtualization pods in your cluster, the `OutdatedVirtualMachineInstanceWorkloads` alert fires. You can view a list of outdated workloads by using the CLI.
```

Comment:
@erkanerol is this correct? Can you teach the reader how to watch for this alert?

Reply:
yes, it is, you can find some info on https://github.com/kubevirt/kubevirt/pull/4686 looking at the `Monitoring and Alerts` section.